taskcluster/app_services_taskgraph/build_config.py

```diff
@@ -10,20 +10,45 @@
 from taskgraph.util.memoize import memoize
 
 
+EXTENSIONS = {
+    'aar': ('.aar', '.pom', '-sources.jar'),
+    'jar': ('.jar', '.pom')
+
+}
+CHECKSUMS_EXTENSIONS = ('.sha1', '.md5')
+
+
 def get_components():
     build_config = _read_build_config()
     return [{
-        'version': build_config['libraryVersion'],
         'name': name,
         'path': project['path'],
-        'artifactId': project['artifactId'],
-        'publications': [{
-            'name': publication['name'],
-            'type': publication['type'],
-        } for publication in project['publications']]
     } for (name, project) in build_config['projects'].items()]
 
 
+def get_version():
+    return _read_build_config()["libraryVersion"]
+
+
+def get_extensions(module_name):
```

Comment:
In AC we only needed the list of extensions returned here. In App-services we also needed the name of the module. Hence, I've enriched a bit the structure to a dictionary where the keys are the `publication name`. Rest of the bits are similar to https://github.com/mozilla-mobile/android-components/blob/8da8237bf9ccf1cf43088b995db5d4ca02c28a69/taskcluster/ac_taskgraph/build_config.py#L37